float32 vEgo
float32 gas
bool gasPressed 
int8 brake
bool brakePressed
float32 steeringAngle
float32 steeringTorque
bool steeringPressed
int8 gearShifter
float32 steeringRate
float32 aEgo
float32 vEgoRaw
bool standstill
bool brakeLights
bool leftBlinker
bool rightBlinker
float32 yawRate
bool genericToggle
bool doorOpen
bool seatbeltUnlatched
bool canValid
float32 steeringTorqueEps
float32 clutchPressed